esx_service/vmdk_ops.py

```diff
@@ -563,8 +564,11 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     Returns None (if all OK) or error string
     """
-
     vm_datastore = get_datastore_name(config_path)
+    result = auth.authorize(vm_uuid, vm_datastore, cmd, opts)
```

Comment:
No, this will not throw ever. It will either return success or an error message. If a VM isn't in a tenant it will be treated  as being in the default tenant and use the current dockvols directory. If the DB is not there, I believe it should do the same thing. The only reason it wouldn't be there is during intialization or if tenants aren't configured since we are going to be putting the DB on shared (VSAN) datastore.